understanding/22/fixed-reference-points.html

```diff
@@ -20,21 +20,21 @@ <h2>Fixed Reference Points Normative Text</h2>
       <blockquote>
          <p class="conformance-level">A</p>
 
-         <p>When a <a>web page</a> or <a>set of web pages</a> is an <a>electronic publication</a> with <a>pagebreak locators</a>, a mechanism is available to navigate to each locator and each locator maintains its place in the flow of content, even when the formatting or platform change.</p>
+         <p>When web content is an <a>electronic publication</a> with <a>page break locators</a> a mechanism is available to navigate to each locator, and each locator maintains its place in the flow of content even when the formatting or platform change.</p>
       </blockquote>
 
       <h3>Definitions:</h3>
       <p><strong>electronic publication</strong>: Content presented as a collection of related articles, document in the form of a book, textbook, magazine article, journal, a single journal article that is presented as having different pages, scholarly journal, or newspaper article in digital format.</p>
-      <p><strong>pagebreak locators</strong> Visual and/or programmatic markers that are arranged in a meaningful sequence to determine the location of a page in relation to others in the set.</p>
+      <p><strong>page break locators</strong>: visual and/or programmatic markers that are arranged in a meaningful sequence to determine the location of a page in relation to others in the set.</p>
```

Comment:
These changes flow from the defined term document so don't need to be fixed in the understanding document as